lib/keypair.ex

```diff
@@ -70,10 +73,9 @@ defmodule Elixium.KeyPair do
   """
   @spec get_priv_from_file(String.t()) :: {binary, binary}
   def get_priv_from_file(pub) do
-    unix_address =
-      :elixium_core
-      |> Application.get_env(:unix_key_address)
-      |> Path.expand()
+      data_path = Application.get_env(:elixium_core, :data_path) |> Path.expand()
+      unix_address = data_path <> Application.get_env(:elixium_core, :unix_key_address)
```

Comment:
Now that we have a `:data_path` I don't think we need a `:unix_key_address` config. We can just have this append "keys" to the `:data_path`